collections/_article/godot-3-6-released.md

```diff
@@ -0,0 +1,345 @@
+---
+title: "Godot 3.6: Released"
```

Comment:
I have yet to review the rest of the blog post, but as I needed to put a link to the announcement in CHANGELOG.md before committing `3.6.stable`, I thought a bit about the title.

I think we can go with the obvious and use this (also renaming the file to `godot-3-6-finally-released`, which is the URL I committed for now in my fork). I'm starting a tentative build now from https://github.com/akien-mga/godot/commit/de2f0f147c5b7eff2d0f6dbc35042a4173fd59be.

We can discuss more tomorrow with Nat and see if she has better ideas comms wise, with the caveat that it would force me to start a new build if we decide to use a different name (but that should be fine, I'm not sure I can manage to release on Monday anyway).
```suggestion
title: "Godot 3.6 finally released!"
```

Reply:
That sounds sensible. Even if Nat wants to make changes, we can orobably keep title the same.